src/api/app/models/project.rb

```diff
@@ -1691,6 +1692,10 @@ def dashboard
     packages.find_by(name: 'dashboard')
   end
 
+  def stageable_requests
+    BsRequest.where(id: BsRequestAction.bs_request_ids_of_involved_projects(id))
```

Comment:
This is already the 
```
target_of_bs_requests
```

assocation, no?

Reply:
YES! hahahah :smile:

Reply:
Changed. Please re-review.